src/shellingham/nt.py

```diff
@@ -106,6 +106,10 @@ def _get_executable(process_dict):
 def get_shell(pid=None, max_depth=6):
     """Get the shell that the supplied pid or os.getpid() is running in.
     """
+    import shellingham.windows as windows
+    # I don't know what the rest of the nt.py thing does.
+    return windows.get_shell(pid, max_depth)
```

Comment:
The `nt` module contains code to detect the shell on NT platforms (IOW non-ancient Windows, technically called [Windows NT](https://en.wikipedia.org/wiki/Windows_NT), which Windows NT the operating system belongs to but is not exactly). So all the code in `windows.py` can simply be added there without a local import like this.

Reply:
Do you want me to replace nt.py altogether? if get_shell is the only public function there, then that is the thing to do.